scripts/common.sh

```diff
@@ -290,8 +290,9 @@ function run_benchmark() {
     local instances=$2
     local inject=$3
     local log_dir="${BENCHMARK_LOGFILE_DIR:-.}/"
-    local duration=${4:-30m}
-    local rate=${5:-800}
+    local duration=${4:-10m}
```

Comment:
30m -> 10m and 800 -> 100, do we still want to change that? Maybe it should be mentioned in commit message or go into separate commit?

Reply:
Fixed, good catch!